src/test/java/com/microsoft/aad/msal4j/ClaimsTest.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+
+public class ClaimsTest {
+
+    @Test
+    public void testClaimsRequest_Format() {
+
+        List<String> values = new ArrayList<>();
+        values.add("urn:mace:incommon:iap:silver");
+        values.add("urn:mace:incommon:iap:bronze");
+
+        ClaimsRequest cr = new ClaimsRequest();
+        cr.requestClaimInUserInfo("given_name", new RequestedClaimAdditionalInfo(true, null, null));
+        cr.requestClaimInUserInfo("email", null);
+        cr.requestClaimInIdToken("acr", new RequestedClaimAdditionalInfo(false, null, values));
+        cr.requestClaimInIdToken("sub", new RequestedClaimAdditionalInfo(true, "248289761001", null));
+
+        Assert.assertEquals(cr.formatAsJSONString(), TestConfiguration.CLAIMS_REQUEST);
+    }
+
+    @Test
+    public void testClaimsRequest_MergeWithClientCapabilities() {
```

Comment:
we need to test claims in actual requests send to STS,
Also we need to test merge logic with claim challenge

Reply:
Added tests merge behavior between claims requests/client capabilities and claims requests/claims challenge, as well as all three together.